genome_designer/main/static/js/contig_view.js

```diff
@@ -38,4 +55,20 @@ gd.ContigView = Backbone.View.extend({
     ));
   },
 
+   /** Puts UI in the loading state. */
+  enterLoadingState: function() {
```

Comment:
If we ever make a common parent view component, this code should be there haha. It's duplicated a million times.